fem/pgridfunc.hpp

```diff
@@ -315,6 +315,13 @@ class ParGridFunction : public GridFunction
        the local dofs. */
    virtual void Save(std::ostream &out) const;
 
+#ifdef MFEM_USE_ADIOS2
+   /** Save the local portion of the ParGridFunction. It differs from the
+       serial GridFunction::Save in that it takes into account the signs of
+       the local dofs. */
+   virtual void Save(adios2stream &out, const std::string &variable_name) const;
```

Comment:
Since this doesn't have the same function signature as the GridFunction::Save, the nvcc compiler repeatedly throws a warning about the GridFunction::Save only being partially overridden in ParGridFunction. The simplest change would be to make them the same to silence that warning.

Reply:
@rcarson3 should be resolved. Good catch.